module.go

```diff
@@ -92,6 +92,14 @@ func CreateModule(smiModule *types.SmiModule) (module SmiModule) {
 	}
 }
 
+func LoadModuleFromStream(modulePath string, data []byte) (string, error) {
+	moduleName := smi.LoadModuleFromStream(modulePath, data)
+	if moduleName == "" {
+		return "", fmt.Errorf("Could not load module at %s", modulePath)
```

Comment:
What is `modulePath` used for in this context?

Reply:
it's passed into GetModuleFromStream() which first checks if "modulePath" has already been loaded, so it does not repeat the work

Reply:
seems this might have been copied from `LoadModule` where `path` seems to imply a path from the filesystem. In the context of loading from stream, I am not sure it makes much sense. Should it be "moduleName"?

Reply:
yes, this function is mostly copied from `LoadModule()` .
When mib files are built into binary, the generated go code provides a function for loading the bytes:
`func Asset(name string)`
Each mib file is loaded by its original file name to this function, which is mapped to the blob data:
`type bintree struct {
	Func     func() (*asset, error)
	Children map[string]*bintree
}
var _bintree = &bintree{nil, map[string]*bintree{
	"MPLS-LDP-MIB.mib": &bintree{mplsLdpMibMib, map[string]*bintree{}},
	"SNMP-COMMUNITY-MIB.mib": &bintree{snmpCommunityMibMib, map[string]*bintree{}},
	"SNMP-FRAMEWORK-MIB.mib": &bintree{snmpFrameworkMibMib, map[string]*bintree{}},
...`